UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -952,12 +973,15 @@ public void Harvest()
 		}
 
 		[Server]
-		protected virtual void Gib()
+		public virtual void Gib()
 		{
-			//TODO: Reimplement
-
-			//never destroy players!
-			_ = Despawn.ServerSingle(gameObject);
+			Death();
+			_ = SoundManager.PlayAtPosition(SingletonSOSounds.Instance.Slip, gameObject.transform.position, gameObject); //TODO: replace with gibbing noise
+			CirculatorySystem.Bleed(GetTotalBlood());
+			foreach(RootBodyPartContainer container in RootBodyPartContainers.ToArray())
+			{
+				container.RemoveLimbs();
```

Comment:
RemoveLimbs() 
destroys the root body part game object, it shouldn't do that since you need that to implant body parts into people

Reply:
Except you don't you completely lost your body and you're now nothing but gibs and blood painting the floor. You lost your entire body.

Reply:
well, it's not Needed then if the body is going to be destroyed

Reply:
Actually no, destroying the body object itself is just a whole can of issues + this solution is the only way we can do proper gibbing without the game throwing errors or doing anything strange. I've tried going through each individual limb and remove them and tried many other solutions and they all were a pain to fix so this is the only way to do proper gibbing right now and I don't think we need to approach this differently since it works as intended and isn't 50+ lines long.

Reply:
just don't destroy the body root part controller , the rest is fine

Reply:
done. I think we're ready to merge now.